scrapers_next/nd/committees.py

```diff
@@ -0,0 +1,90 @@
+from spatula import URL, HtmlListPage, HtmlPage, XPath
+from openstates.models import ScrapeCommittee
+import re
+
+
+class SubcommitteeFound(BaseException):
+    def __init__(self, com_name):
+        super().__init__(
+            f"Scraper has no process for ingesting subcommittee classification: {com_name}"
+        )
+
+
+class CommitteeDetail(HtmlPage):
+    def process_page(self):
+        com = self.input
+        staff = {"staff": []}
+        members = XPath('//div[@class="member-wrapper"]').match(self.root)
+        for each_member in members:
+            member_detail = each_member.text_content().strip()
+            member_detail = [x.strip() for x in member_detail.splitlines() if x.strip()]
+
+            if len(member_detail) == 5:
+                role = member_detail[-1]
+            else:
+                role = "Member"
+            name = [
+                x.text_content()
+                for x in each_member.findall("div", {"class": "strong member-name"})
+            ]
+
+            if len(name) == 1:
+                name = re.sub("\\s+", " ", name[0]).strip()
+                staff["staff"].append({"name": name, "role": "staff"})
+                continue
+            elif len(name) == 3:
+                name = f"{name[0]}{name[1]}"
+                name = re.sub("\\s+", " ", name).strip()
+                staff["staff"].append(
+                    {"name": name, "role": "Committee Citizen Members"}
+                )
+                continue
+            else:
+                name = member_detail[1]
+            com.add_member(name, role)
+        com.extras = staff
+        com.add_source(
+            self.source.url,
+            note="Committee Details page",
+        )
```

Comment:
My comment was actually requesting that the `add_link()` method be used **in addition** to the `add_source()` method, for the committee detail page.

So, you should:
- add BACK IN the `com.add_source(self.source.url, note="Committee Details page")` like you had before
- BUT ALSO ADD `com.add_link(self.source.url, note="homepage")` as requested in the comment

This will be the case for this and all future scrapers. It is a requirement within our data pipeline that each committee MUST have a `link` added for the detail page, and that the note for that link must be `"homepage"`.